### ISSUE TITLE
Youtube 'Watch8' UI breaks script

### ISSUE THREAD

**Issue by Cyberbeing** [rossy/youtube-video-download#18]
It appears Youtube has been doing a random lottery rollout of a new 'Watch8' UI which eliminates the bar with text buttons of 'Watch7' and replaces it with tiny square icons without text. For obvious reasons, this breaks the 4.1.2 version of the script. The expected location which the script would normally add the Download button no longer exists in the new UI.

Under the assumption that this new 'Watch8' UI eventually becomes the default, you may want to get a headstart and look into a workaround or fix this problem in your next update of the script.

**Comment by AGresvig** [comment]
Yes, I'm seeing it too. Example video: https://www.youtube.com/watch?v=p9zUK8pOEgQ

**Comment by Cyberbeing** [comment]
Linking videos likely won't do any good for reproducing this, since Youtube is randomly tagging browsers on first visit to use either the watch7 or watch8 UI on all videos.

If you trigger this at all, chances are it will only occur on a single browser install out of many on your system. You may have to try a few times, but if you wipe your cache and possibly cookies as well, it should be possible to get back on the watch7 UI for the time being.

**Comment by AGresvig** [comment]
Ah OK I see, thanks for the info.

**Comment by Cyberbeing** [comment]
Just noticed today that Youtube seems to have rolled out a new revision of Watch8 UI (www-core-vfl0pJopz.css dated 8/27/2014), which this time actually seems to be backwards compatible with previous UI elements. Assuming this ends up being the final version, this script can be easily fixed by doing a find/replace of watch7 to watch8.

**Comment by olecom** [comment]
@svtux, it works, thanks!

**Comment by svtux** [comment]
@olecom Пожалуйста! You are welcome!
